tests/cfg.rs

```diff
@@ -247,7 +247,8 @@ fn ignore_version_from_other_platform() {
     assert_that(p.cargo_process("build"),
                 execs().with_status(0).with_stderr("\
 [UPDATING] registry [..]
-[DOWNLOADING] [..]
+[DOWNLOADING] foo [..]
+[DOWNLOADING] foo [..]
```

Comment:
Argh I forgot about this! This is actually a pretty crucial feature today where builds on Linux don't download Windows-only dependencies.